go/tasks/plugins/array/awsbatch/client.go

```diff
@@ -9,6 +9,11 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/pkg/errors"
```

Comment:
nit, rearrange imports to be alphabetical and grouped by repository

```suggestion
	a "github.com/aws/aws-sdk-go/aws"
	"github.com/aws/aws-sdk-go/aws/request"
	"github.com/aws/aws-sdk-go/service/batch"

	"github.com/flyteorg/flyteplugins/go/tasks/aws"
	pluginUtils "github.com/flyteorg/flyteplugins/go/tasks/pluginmachinery/utils"
	definition2 "github.com/flyteorg/flyteplugins/go/tasks/plugins/array/awsbatch/definition"

	"github.com/flyteorg/flytestdlib/utils"
	"github.com/flyteorg/flytestdlib/logger"

	structpb "github.com/golang/protobuf/ptypes/struct"
	
	"github.com/pkg/errors"
```